features/bevragen/persoon/partner/aangaan-huwelijk-partnerschap/dev/aangaanHuwelijkPartnerschap-gba.feature

```diff
@@ -0,0 +1,44 @@
+# language: nl
+
+Functionaliteit: aangaan huwelijk/partnerschap bij actuele of ontbonden relatie
+
```

Comment:
toelichten dat bij ontbonden partner de sluiting/aangaan in historische rij zit.

Reply:
ook scenario toevoegen waarbij ontbinding gewijzigd is (aangaan halen uit volgnummer 2)